doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -476,7 +476,7 @@ nRF RPC libraries
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* Removed the unused SDFW services ``echo_service``, ``reset_evt_service`` and ``sdfw_update_service``.
```

Comment:
```suggestion
* Removed the unused SDFW services ``echo_service``, ``reset_evt_service``, and ``sdfw_update_service``.
```

Reply:
This entry is a bit suspicious, as we don't seem to mention these services in any of the NCS libraries.